workflow/infrastructure-monitoring/container-monitoring.mdx

```diff
@@ -1,73 +1,131 @@
 
-![](/images/infra-monitoring/c1.png)
+Quickly isolate and drill into blockers within your teams Infrastructure stack, monitor your cluster, node, and pod data in one place, and effortlessly manage and review your overall Infrastructure utilization cluster by cluster.
 
-### Container ID
-This shows the id of the container and IDs are randomly generated and are typically 12 characters long.
+# Key Concepts
 
-### Container Name
-This shows the name of the container and the name is a user-friendly identifier that is assigned to a container.  
+### Host List
 
-### Host
-The host can be a physical or virtual machine running any operating system, such as Linux, Windows, or macOS.  
+Host List provides insight into your Kubernetes orchestration data, database usage, system-wide infrastructure utilization and more. It is the highest level compoenent overview, providing a top-down approach for diagnosing, monitoring, and maintaining your stack.
 
-### Status
-The container status refers to the current state of a container. The container status states include:
+### Containers
 
--   **Running**: The container is currently running and its main process is active.
--   **Restarting**: The container is in the process of restarting after being stopped or restarted.
--   **Exited**: The container has stopped running and its main process has exited.
--   **Dead**: The container has stopped running due to an error or crash, and its main process is no longer active.
+Containers provide visibility into the health and performance of your containerized environments. This section does not include containers that are part of an orchestration tool (e.g. Kubernetes). 
 
-### CPU Usage
-This shows the percentage of CPU used. The CPU usage of a container refers to the percentage of the host system's CPU resources that are currently being used by the container's processes.
+### Processes
 
-### RAM Usage
-This shows the percentage of RAM used. The container RAM usage in percentage refers to the proportion of allocated memory resources that a container is currently using on the host system.
+Processes are managed within your containers to ensure your Infrastructure is effectively monitored, maintained, and optimized for performance, reliability, and operability.
 
-### Started Ago
-This indicates when the container was started. when a container is started, and records the time it was started and saves this information as metadata associated with the container.
+### Kubernetes
 
-Filters
--------
-We have many options for data filters. Like selecting any date and searching container by a container name, container id, container image, etc. Also in the sidebar section, we also have multiple check options available to filter the containers list data.
+Kubernetes is a container orchestration platform that automates the deployment, scaling, management, and operation of containerized applications. Effortlessly personalize your k8s data visualization experience.
```

Comment:
Edit this section per the framework shared above. The intro sentence is great, now we need the how do we collect it and what to do with it sentences.

Reply:
also checking on this with Tejas

Reply:
Use Kubernetes not K8s

Run our MW agent as a Kubernetes daemonset.

that thing collects the data and sends it to the MW agent.